docs/source/user-guide/configuration/use-condarc.rst

```diff
@@ -522,6 +522,28 @@ The CONDA_ENVS_PATH environment variable overwrites this setting:
   ``set CONDA_ENVS_PATH=C:\Users\joe\envs;C:\Anaconda\envs``
 
 
+Specify package directories (pkgs_dirs)
+---------------------------------------
+
+Specify directories in which packages are located. If this
+key is set, the root prefix ``pkgs_dir`` is not used unless
+explicitly included.
+
+EXAMPLE:
+
+.. code-block:: yaml
+
+  pkg_dirs:
```

Comment:
please double check this.  Is it pkg_dirs, pkgs_dirs, pkgs_dir?  Extra s's matter a lot.  You have a lot of inconsistency here.

Reply:
It is pkgs_dirs. Fixed those mistakes. 
Thank you!